packages/grafana-data/src/types/config.ts

```diff
@@ -64,6 +64,8 @@ export interface UnifiedAlertingConfig {
   minInterval: string;
   // will be undefined if alerStateHistory is not enabled
   alertStateHistoryBackend?: string;
+  // will be undefined if implementation is not "multiple"
```

Comment:
@alexweav can I bother you to double check this assumption? Testing it locally I was only getting the primary for the `multiple` backend

Reply:
This is correct as well.